charts/rancher-logging/0.0.1/Chart.yaml

```diff
@@ -6,7 +6,8 @@ version: 0.1.0
 appVersion: v1.1.3
 home: https://www.fluentd.org/
 sources:
-- https://www.fluentd.org/
+  - https://www.fluentd.org/
 maintainers:
-- name: Michelia
-  email: support@rancher.com
+  - name: Michelia
+    email: support@rancher.com
+kubeVersion: <1.21.0-0
```

Comment:
same comment as above, is it possible that someone could be using Logging v1 in an imported 1.21+ cluster today?

Reply:
same comment for all the prior logging versions too